global/concourse/templates/worker-policy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: policy/v1beta1
```

Comment:
Why do we need to add this? I don't see it being used.
Same for the worker-role and service account

Reply:
I removed the pod policy, serviceaccount/roles might be used in future.